tsconfig.shared.json

```diff
@@ -6,7 +6,7 @@
     "jsx": "preserve",
     "target": "esnext",
     "module": "esnext",
-    "lib": ["dom", "esnext"],
+    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
@swissspidy, this seems logical. Do you see any reason why it shouldn't be there?

Reply:
Haven't fully looked into [the `lib` option](https://www.typescriptlang.org/tsconfig#lib) but yeah this seems reasonable